src/invidious/routes/preferences.cr

```diff
@@ -140,6 +140,10 @@ module Invidious::Routes::PreferencesRoute
     notifications_only ||= "off"
     notifications_only = notifications_only == "on"
 
+    previous_page_button = env.params.body["previous_page_button"]?.try &.as(String)
+    previous_page_button ||= "off"
+    previous_page_button = previous_page_button == "on"
+
```

Comment:
As mentioned in other PRs, I'm not found of adding more config options, especially when we don't have any way in the UI to explain in more details what a config option does ("Previous page button (requires Javascript)" is not very explicit).

I'd prefer if that feature was always enabled for clients with JS.

In essence, show a "first page" button by default, and if some continuations are saved, show a "previous page" button.